src/test/java/com/example/demo/repository/BookRepositoryTest.java

```diff
@@ -0,0 +1,28 @@
+package com.example.demo.repository;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.example.demo.model.Book;
+import com.example.demo.repository.book.BookRepository;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+class BookRepositoryTest {
+    @Autowired
+    private BookRepository bookRepository;
+
+    void findById_GivenExistingBook_ShouldReturnBook() {
```

Comment:
Don’t test the methods of the crud repository. You should test your custom methods